src/main/java/org/sqlite/core/CoreDatabaseMetaData.java

```diff
@@ -184,6 +184,19 @@ protected String escape(final String val) {
         return buf.toString();
     }
 
+    /**
+     * Returns empty value or escaped schema prefix
+     * @param schema schema name
+     * @return The SQL escaped schema name with dot or empty string
+     */
+    protected String getSchemaPrefix(String schema) {
```

Comment:
Maybe it should be called `prependSchema` and accept both schema name and * anything * that goes after it?

This way, we can avoid pointless concatenations if the schema is not defined